### PR TITLE
Fix bug where profile search can't be closed

### PR DESCRIPTION
**Pull request by akalin-keybase**
Happens when instantiated via 'Browse people' button.

### CODE REVIEW

File: shared/profile/search/container.js [hunk 1]
```diff
@@ -1,21 +1,45 @@
 // @flow
 import Search from '.'
 import {createShowUserProfile} from '../../actions/profile-gen'
-import {connect} from '../../util/container'
+import {connect, type RouteProps} from '../../util/container'
 
-type OwnProps = {|onClose: () => void|}
+// Either onClose is passed in from bar.js or we're instantiated via a
+// route.
+type OwnProps = {|onClose: () => void|} | RouteProps<{}, {}>
```
Comment (akalin-keybase) [inline shared/profile/search/container.js]: @cjb Ah, I see what you mean here. I only need it for `ownProps.navigateUp`, but maybe I should just use `dispatch(RouteTreeGen.createNavigateUp())`?

Comment (cjb) [reply shared/profile/search/container.js]: yup!

Comment (akalin-keybase) [reply shared/profile/search/container.js]: Fixed!

**Comment by akalin-keybase** [comment]
PTAL

**Comment by akalin-keybase** [comment]
bump!

**Pull request merged**